addons/web/static/src/views/list/list_renderer.scss

```diff
@@ -372,11 +372,11 @@
         }
     }
 
-    .o_data_row.o_selected_row > .o_data_cell.o_required_modifier:not(.o_readonly_modifier) {
+    .o_data_row.o_selected_row > .o_data_cell.o_required_modifier:not(:is(.o_readonly_modifier)) {
```

Comment:
`:is` != `:has`

In this case, you basically rewritten the same selector as before :shrug:

Reply:
I know, But `:is` is supported in Firefox and almost every browser that's why I change it And I also added a new commit to add `o_readonly_modifier` on parent(`td`) also same as `o_required_modifier` and `o_invalid_cell` so it will fix issue without changing scss.
could you check?
Thanks!! :slightly_smiling_face: 
cc: @IT-Ideas @brboi